examples/external-data/src/mock-external-data-service/service.ts

```diff
@@ -286,9 +286,9 @@ export async function initializeExternalDataService(props: ServiceProps): Promis
 				.json({ message: "Missing parameter externalTaskListId in request url" });
 		}
 		externalDataSource.fetchData(externalTaskListId).then(
-			(response) => {
-				// eslint-disable-next-line @typescript-eslint/no-base-to-string
-				const responseBody = JSON.parse(response.body.toString()) as Record<
+			async (response) => {
```

Comment:
This _looks_ fine to me. But mixing `async` with non-`async` continuations is generally frowned upon AFAIK. It might be better to make this non-async and use a `.then` pattern inside.